pages/projects.tsx

```diff
@@ -17,6 +17,7 @@ const Projects = () => {
       <Navbar active={'projects'} />
       <main className="flex flex-col w-full mt-20">
         <Toaster />
+        <AuthModal />
```

Comment:
could you make the auth modal title customisable? in the issue #82 if you see we have agreed to put the title as 'login to continue'

Reply:
Yeah, It should be like that.